StackUsingLinkedList.java

```diff
@@ -0,0 +1,55 @@
+import java.io.*;
+public class StackUsingLinkedList{
+       public static class Node{
+		          int data;
+		          Node next;
+		   
+		         Node (int d){
+			          data=d;
+			          next=null;
+		          }
+	   }
+	  public static Node top;
+	  
+    public static void push(int x){
+		         Node newNode=new Node(x);   
+		         if(top==null){
+				        newNode.next=null;
+           
+			        }	
+			      else {
+				      newNode.next=top;
+			      }
+			top=newNode;
+	   }
+	   public static void pop(){
+		        if(top==null){
+			            System.out.println("The Stack is empty\n");
+		         }
+		        else{
+			            top=top.next;
+		        }
+	    }
+		public static void print(Node top) {
```

Comment:
@atulya0622 , wrong implementation...I asked u to learn about toString() method ...u don't usually print a stack , but i asked u to so that u can learn about toString()...Implement toString() in the linkedlist and print the object